sql/core/src/main/scala/org/apache/spark/sql/DataFrameWriterV2.scala

```diff
@@ -204,6 +206,7 @@ final class DataFrameWriterV2[T] private[sql](table: String, ds: Dataset[T])
    */
```

Comment:
Should we write it about `AnalysisException` and update the docs above? maybe at least just for the sake of matching.

Reply:
+1 for @HyukjinKwon 's suggestion.

Reply:
I thought about this, and decided to skip because AnalysisException is thrown for whatever case even without this case. We couldn't explain all these things. I'll add it if we prefer to explain it at least for explicit case.